src/seq/UndoBridge.cpp

```diff
@@ -45,25 +45,25 @@ class SeqAction : public ::rack::history::ModuleAction
         SequencerPtr ret;
         Module* module = dynamic_cast<Module *>(APP->engine->getModule(moduleId));
         if (!module) {
-            fprintf(stderr, "error getting module in undo\n");
+            WARN("error getting module in undo, id = %I64d", moduleId);
```

Comment:
switched to using the standard VCV log stuff like I usually do. Also printed out the id (to help debug).